REVIEW.md

```diff
@@ -0,0 +1,257 @@
+# Reviewer Guidelines
+
```

Comment:
I'd add a note at the top to state when these guidelines go into effect.

```suggestion

*Effective: [add merge date]*

```

Reply:
I'd advise against this, as manually duplicating the metadata Git tracks automatically and GitHub displays prominently is widely considered a bad practice, as it adds overhead, imprecisely duplicates the canonical source of truth, and has the potential to get out of sync and mislead users, without any corresponding benefit.

The accurate merge date/time of the current version, localized to the user's regional format and timezone, is prominently displayed along with the other key metadata in a banner above the file, and the full history of when different versions went into effect and what changed is only a click away to anyone interested in this information.

Furthermore, the code of conduct and the other policies that see don't have this sort of header, and it comes across to me as rather bureaucratic, not exactly the spirit that FOSS projects typically embody.

Reply:
My concern is that people may look at existing PRs and be confused why this document says one thing and PRs do another. Yes, the date is stored by git, I just don't think it's exposed. I think it could be confusing in the transition.

This is not a blocker for me, though. We don't 100% need to include this.

Reply:
Actually, it is exposed, as mentioned above. Here's how it looks for the code of conduct:

![image](https://user-images.githubusercontent.com/17051931/141172973-eb07b41a-916b-4827-b80f-37ba8dc98afe.png)

In addition to the issues discussed previously, there was a recent thorny incident with a Quansight blog post that didn't get its publish date set correctly in the source on merging (since it happen right before), so even if someone remembers at some point to set the date, it makes merges much more tricky and easy to forget to update (especially since it either requires the author and the commiter to coordinate synchronously, or the committer to be a maintainer and (force) push the author's branch). And unlike a blog post, this dance will need to happen every time every time it is updated.

Reply:
@CAM-Gerlach, I think is important to add the start date to keep track of when it started being effective in the repo. Also, I think we will never have to update it again, at least if we do change a lot this document.

Reply:
Yeah, but as mentioned Git already tracks that and it is displayed just above the rendered document, and if we keep the date static, it may lead to the misleading impression that the _current_ version was published/effective on that date, which (particularly as time goes on and we revise it) may have substantially different requirements than this original.

Reply:
I don't think GitHub's UI puts it in a very visible place and there is always the chance that this document gets hosted outside of GitHub (though it's not linked to the docs right now). 

Still, I don't think this is critical so I'll close the suggestion. Thanks!